pkg/procmatch/gen/generate_catalog.go

```diff
@@ -0,0 +1,87 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+type Manifest struct {
+	DisplayName  *string  `json:"display_name"`
+	MetricPrefix *string  `json:"metric_prefix"`
+	Name         *string  `json:"name"`
+	Signatures   []string `json:"process_signatures"`
+}
+
+const codeTemplate = `// Code generated by go generate; DO NOT EDIT.
+package procmatch
+
+var DefaultCatalog IntegrationCatalog = IntegrationCatalog{ {{ range $_, $manifest := . }}
+	IntegrationEntry{
+		DisplayName: 	"{{$manifest.DisplayName}}",
+		MetricPrefix: 	"{{$manifest.MetricPrefix}}",
+		Name: 			"{{$manifest.Name}}",
+		Signatures: 	[]string{ {{ range $_, $sig := $manifest.Signatures }}
+			"{{$sig}}",{{end}}
```

Comment:
nit: played a bit with the `go generate` command and I noticed it generated a file with weird indentation, it appears the template here was changed since you generated the `default_catalog.go`, could you fix the template? (for example: unnecessary tab before `"{{$manifest.Name}}"`